includes/admin.form.inc

```diff
@@ -127,23 +132,25 @@ function islandora_video_admin_submit($form, &$form_state) {
  * Form validation ensuring the selected ffmpeg options are available.
  */
 function islandora_video_admin_validate($form, &$form_state) {
-  // Ensure the mp4 audio codec is enabled.
-  $raw_value = $form_state['values']['islandora_video_mp4_audio_codec'];
-  if ($raw_value) {
-    if (preg_match('/[^0-9a-zA-Z_-]/', $raw_value) === 1) {
-      form_set_error('', "The value entered for MP4 audio codec contains forbidden characters.");
-      return;
+  // Ensure the mp4 audio codec is present if MP4 derivative is enabled.
+  if ($form_state['values']['islandora_video_make_mp4_locally'] == 1) {
```

Comment:
Minor: This works++, but for clarity maybe compare $form_state['values']['islandora_video_make_mp4_locally'] == TRUE since the default value is true at https://github.com/Islandora/islandora_solution_pack_video/pull/142/files#diff-0cdd19abe14e51fe9e7f061ec72c86d3R53

Reply:
Fixed with 3904cd65e7b96ab1439b094083754be20e37d357.